tests/integration/components/sl-translate-test.js

```diff
@@ -0,0 +1,128 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-translate', 'Integration | Component | sl translate', {
+    beforeEach() {
+        this.container.lookup( 'service:sl-translate' )
+            .setDictionary(
+                Ember.Object.create({
+                    'SIMPLE_KEY': 'I have been translated',
+                    'SINGULAR_KEY': 'View my family',
+                    'PLURAL_KEY': 'View my families',
+                    'REPLACED_KEY': 'I have replaced {0} and {1}'
+                })
+            );
+    },
+
+    integration: true
+});
+
+test( 'Simple Key Translation', function( assert ) {
+    assert.expect( 1 );
+
+    this.render( hbs`
+        {{sl-translate key="SIMPLE_KEY" }}
+    ` );
+
+    assert.equal(
+        this.$().text().trim(),
+        'I have been translated',
+        'Simple Key was translated'
+    );
+});
+
+test( 'Single/Plural Keys', function( assert ) {
+    assert.expect( 2 );
+
+    this.render( hbs`
+        {{sl-translate key="SINGULAR_KEY" pluralKey="PLURAL_KEY" pluralCount="1"}}
+    ` );
+
+    assert.equal(
+        this.$().text().trim(),
+        'View my family',
+        'Singular Key was correctly returned'
+    );
+
+    this.render( hbs`
+        {{sl-translate key="SINGULAR_KEY" pluralKey="PLURAL_KEY" pluralCount="3"}}
+    ` );
+
+    assert.equal(
+        this.$().text().trim(),
+        'View my families',
+        'Plural Key was correctly returned'
+    );
+});
+
+test( 'Replaced Values In Keys', function( assert ) {
+    assert.expect( 2 );
+
+    const replacedKey = 'I have replaced {0} and {1}';
+
+    this.set( 'replacedKey', replacedKey );
+
+    this.render( hbs`
+        {{sl-translate key=replacedKey}}
```

Comment:
Use `REPLACED_KEY` FROM `beforeEach()` instead.  Same for other occurrences in this test, and the assignment to `const`.